Gemfile

```diff
@@ -1,15 +1,19 @@
 source 'https://rubygems.org'
 
+gem 'listen'
+gem 'rouge'
+gem 'execjs'
+gem 'therubyracer'
+gem 'html-proofer'
+gem 'liquid-c'
```

Comment:
This fails to install for me on gLinux. Did you do anything special to get this installed?

```
cc1: error: unrecognized command line option ‘-Wno-self-assign’ [-Werror]
cc1: error: unrecognized command line option ‘-Wno-constant-logical-operand’ [-Werror]
cc1: error: unrecognized command line option ‘-Wno-parentheses-equality’ [-Werror]
cc1: all warnings being treated as errors
make: *** [Makefile:242: block.o] Error 1

make failed, exit code 2
```

Reply:
I installed liquid-c through the normal GEM/bundler installation process using RVM to manage jekyll versions.

Reply:
Is RVM easy to setup? Can you provide any steps/resources for how to set this up locally? Our [website](https://forsetisecurity.org/docs/latest/develop/website.html) and internal docs do not mention RVM. I can update these docs, just need to know what to do.

Reply:
@gkowalski-google Looks like RVM is not being used. @mirons-google updated ruby version in `.ruby-version` file.

Reply:
@gkowalski-google Were you able to install `liquid-c` on your local? I was able to do so by pulling the latest `Gemfile` and by running `bundle update`. Looks like there are no other breaking changes.

Reply:
@red2k18 I tried again, and I still get the same error. I tried with the latest gemfile and `bundle update` along with `gem install liquid-c -v '4.0.0' --source 'https://rubygems.org/'`. This might be dependent on OS. I believe [Travis is using RVM](https://docs.travis-ci.com/user/languages/ruby/#specifying-ruby-versions-and-implementations) and switching to this locally would probably resolve my issue. I don't want to hold up this PR if it's ready, my only concern is if this will cause issues for the next release engineer if they run into the same issue as me. @kevensen Can you try running `gem install liquid-c -v '4.0.0' --source 'https://rubygems.org/'` and see if you get this error?

Reply:
@gkowalski-google I tried @red2k18 instructions and it worked.  I think you have to run `bundle install`

Reply:
I have tried that, several times, no dice. Will have to fix my issue later.